server/application-server/pom.xml

```diff
@@ -29,6 +29,7 @@
 	</scm>
 	<properties>
 		<java.version>21</java.version>
+		<app.profiles>local,dev</app.profiles>
```

Comment:
We also have `prod`, am I right? At least that is what I'm using in the docker compose file `SPRING_PROFILES_ACTIVE=prod`

Reply:
Yes, indeed. Via the command I always had trouble selecting a profile though, so adding it here would auto-overwrite all my values with the prod-placeholders.

Reply:
I see, on Artemis the default profile is also `dev` 👍